documentation/developers.md

```diff
@@ -30,6 +30,32 @@ Code sample:
       v1-status-callback)
     context))
 ```
+
+
+## Implementing Your Status Function
+The puppetlabs.trapperkeeper.services.status.status-core namespace contains
+some utilities to aid in the implementation of your status functions.  In
+particular, the `level->int` function defines an ordering for status levels as
+`:critical < :info < :debug`, and the `level>`, `level<`, `level>=` and `level<=`
+functions can be used to compare status levels.  These are especially useful in
+conjunction with the `cond->` macro from clojure.core.  Here's an example of how
+a status function might be implemented to utilize the `level>=` function:
+```clj
+(require '[puppetlabs.trapperkeeper.services.status.status-core :as status-core])
+
+(defn my-status
+  [level]
+  (let [level>= (partial status-core/level>= level)]
+    {:is-running true
+     :status (cond-> {:this-is-critical "foo"}
+               (level>= :info) (assoc :bar "bar"
+                                      :baz "baz")
+               (level>= :debug) (assoc :x "x"
+                                       :y "y"
+                                       :z "y"))}))
+```
```

Comment:
I agree with @cprice404, and I think it would be easier to add these back in if their needed down the line, but right now it doesn't seem like their really needed.